db/seeds/playbooks/spooky.seeds.rb

```diff
@@ -0,0 +1,274 @@
+# frozen_string_literal: true
+
+after :playbook do
+    @spooky = Playbook.find_by! name: 'The Spooky'
+  
+    ######
+    # Moves
+    # ratings: charm: 0, cool: 1, sharp: 2, tough: 3, weird: 4
+    ######
+    [{
+      name: 'Telepathy',
+      type: 'Moves::Descriptive',
+      playbook_id: @spooky.id,
+      description: 'You can read people’s thoughts and put words 
+      in their mind. This can allow you to investigate a mystery 
+      or read a bad situation without needing to actually talk. 
+      You an also manipulate someone without speaking. You still 
+      roll moves as normal, except people will not expect the 
+      weirdness of your mental communication.'
+    }
```

Comment:
```suggestion
    },
```
Missing comma!

Reply:
This has been fixed along with other proofreading items.